va/validation-authority.go

```diff
@@ -311,7 +383,7 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	}
 	defer conn.Close()
 
-	// Check that ZName is a dNSName SAN in the server's certificate
+	// Check that ZName is a DNSName SAN in the server's certificate
```

Comment:
Leave the original capitalization here. It's not an error: that's just how field names are capitalized in the X.509 RFC: https://www.rfc-editor.org/rfc/rfc2459.txt